docs/source/tutorials/oregon.rst

```diff
@@ -0,0 +1,688 @@
+
+    plt.tight_layout()
+    plt.show()
+
+The analysis produces the following local distribution treatment effects visualization:
+
+.. image:: ../_static/oregon_ldte_costs_comparison.png
+   :alt: Oregon Health Insurance Experiment LDTE Analysis
+   :width: 800px
+   :align: center
+
+**LDTE Interpretation**: The positive LDTE values indicate that Medicaid assignment increases the cumulative probability of individuals having emergency department costs at or below each threshold among compliers (those who enroll when selected). This suggests that while Medicaid increases overall ED utilization, it may also help contain costs for some individuals who actually enroll.
+
+**Statistical Significance**: Both simple and ML-adjusted local estimators show similar patterns, providing robust evidence that Medicaid assignment has significant distributional effects on emergency department costs for compliers. The confidence intervals indicate that these effects are statistically significant across most cost levels.
+
+**Non-Compliance Considerations**: The LDTE analysis accounts for the fact that not all individuals assigned to treatment actually enrolled in Medicaid, providing a more accurate estimate of the effects for those who actually receive treatment when assigned.
+
+Cost Analysis with Local PTE
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    # Compute Local Probability Treatment Effects
+    lpte_simple, lpte_lower_simple, lpte_upper_simple = simple_local_estimator.predict_lpte(
+        target_treatment_arm=1,  # Z=1 Selected for treatment (Enrolled)
+        control_treatment_arm=0,  # Z=0 Not selected for treatment (Not enrolled)
+        locations=[-1] + outcome_ed_costs_locations
+    )
+
+    lpte_ml, lpte_lower_ml, lpte_upper_ml = ml_local_estimator.predict_lpte(
+        target_treatment_arm=1,  # Z=1 Selected for treatment (Enrolled)
+        control_treatment_arm=0,  # Z=0 Not selected for treatment (Not enrolled)
+        locations=[-1] + outcome_ed_costs_locations
+    )
+
+    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(15, 6))
+
+    # Simple local estimator
+    plot(outcome_ed_costs_locations[1:], lpte_simple, lpte_lower_simple, lpte_upper_simple,
+        chart_type="bar",
+        title="Effects of Emergency Department Costs (Simple Local Estimator)",
+        xlabel="Emergency Department Costs", ylabel="Local Probability Treatment Effect",
+        color="purple",
+        ax=ax1)
+
+    # ML-adjusted local estimator
+    plot(outcome_ed_costs_locations[1:], lpte_ml, lpte_lower_ml, lpte_upper_ml,
+        chart_type="bar",
+        title="Effects of Emergency Department Costs (ML-Adjusted Local Estimator)",
+        xlabel="Emergency Department Costs", ylabel="Local Probability Treatment Effect",
+        ax=ax2)
+    plt.tight_layout()
+    plt.show()
+
+The Local Probability Treatment Effects analysis produces the following visualization:
+
+.. image:: ../_static/oregon_lpte_costs_comparison.png
```

Comment:
Maybe we can increase the size of each bin. Currently there are too many bins in the graph.